pkg/externalbackups/types.go

```diff
@@ -0,0 +1,9 @@
+package externalbackups
+
+const (
+	// S3Type represents the AWS S3 backup typ.
+	S3Type = "s3"
+
+	// GCSType represents the Google cloud storage backup typ.
+	GCSType = "gcs"
+)
```

Comment:
Is there a reason to have this here within `pkg` compared to `central/externalbackups/plugins/types`?

The latter feels like the more natural choice.